File: PHP/1020 - Idade em dias.php
<?php  

	$dias = (int)readline();

	$ano = (int) ($dias/365); 
	$mes = (int) ($dias%365/30);
	$dias = (int) ($dias%365%30);

	echo "$ano ano(s)\n$mes mes(es)\n$dias dia(s)\n";

?>

File: CS/1020 - Idade em dias.cs
using System;

class URI
{
	static void Main()
	{
		int anos, meses, dias;

		dias = int.Parse(Console.ReadLine());

		anos = dias/365;
		meses = dias%365/30;
		dias = dias%365%30;
	
		Console.WriteLine(anos + " ano(s)");
		Console.WriteLine(meses + " mes(es)");
		Console.WriteLine(dias + " dia(s)");
	}
}